Gemfile

```diff
@@ -3,7 +3,15 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.6.5'
+def ruby_version
```

Comment:
Might be worth updating the README to include `asdf` - some existing copy here - https://github.com/DFE-Digital/teacher-training-api#native-1